model/src/main/proto/profile.proto

```diff
@@ -53,6 +62,9 @@ message Profile {
 
   // Represents user selected app-language.
   AppLanguage app_language = 12;
+
+  // Determine admins settings.
+  AdminSettings adminSettings = 13;
```

Comment:
As discussed earlier, the AdminSettings should be a part of `ProfileDatabase` instead of `Profile` because assuming there are 10 profiles, so only one will have values for `AdminSettings` and rest will have null value, which is not that clean.

Reply:
Can we set admin settings for only profile Id with `0`.